packages/edit-site/src/components/editor/use-editor-title.js

```diff
@@ -22,6 +22,10 @@ function useEditorTitle( postType, postId ) {
 			const { getEditedEntityRecord, hasFinishedResolution } =
 				select( coreStore );
 
+			if ( ! postId ) {
```

Comment:
I think for the home page of the site editor, there's no postId (and other pages). Won't this have an impact there?

Reply:
I think when you are on data view pages, the title does not update, but that's the case in trunk too?

Reply:
Btw, it's currently not erroring because `getEditedEntityRecord` returns `false` if it's not loaded yet, which `getTemplateInfo` is destructuring.

Reply:
I think you're right.